lib/app/main.dart

```diff
@@ -1,19 +1,32 @@
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
 import 'home/home_view.dart';
+import 'login/login_view.dart';
 
 void main() => runApp(MyApp());
 
 class MyApp extends StatelessWidget {
   // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
+
+    SystemChrome.setSystemUIOverlayStyle(
+        SystemUiOverlayStyle(statusBarColor: Colors.red));
+
     return MaterialApp(
-      title: 'Flutter Demo',
-      theme: ThemeData(
-        primarySwatch: Colors.blue,
-      ),
-      home: HomePage(title: 'Flutter Demo Home Page'),
-    );
+        title: 'Flutter Demo',
+        theme: ThemeData(
+          brightness: Brightness.dark,
+          primaryColor: Colors.red,
+          accentColor: Colors.redAccent,
+          primarySwatch: Colors.red,
+          fontFamily: 'Poppins',
+        ),
+        home: LoginPage(title: 'Flutter Demo Home Page'),
+        debugShowCheckedModeBanner: false,
+        routes: <String, WidgetBuilder>{
+          '/home': (BuildContext context) => new HomePage(),
```

Comment:
Create a `Constants` class in the `App` layer and store the named routes as strings in there to minimize errors.
So `app/utils/constants.dart`.